lib/tableau/extensions/rss_extension.ex

```diff
@@ -36,6 +36,8 @@ defmodule Tableau.RSSExtension do
   end
 
   def run(%{site: %{config: %{url: url}}, posts: posts, extensions: %{rss: %{config: rss}}} = token) do
+    out = Map.get(token, :out, "_site")
```

Comment:
i'm actually fixing this is another PR i just opened (#125), so please remove any changes to the rss extension from this patch